extension/src/cli/dvc/discovery.ts

```diff
@@ -12,22 +8,33 @@ import {
   getConfigValue,
   setUserConfigValue
 } from '../../vscode/config'
-import { getPythonBinPath } from '../../extensions/python'
 import { getFirstWorkspaceFolder } from '../../vscode/workspaceFolders'
 import { delay } from '../../util/time'
 import { SetupSection } from '../../setup/webview/contract'
 
+const warnWithSetupAction = async (
```

Comment:
Current possible options are:

**Version not found:**
`An error was thrown when trying to access the CLI.`
buttons: Setup, Don't Show Again

**Version can not be verified:**
`The extension cannot initialize as we were unable to verify the DVC CLI version.`
Button: Setup

**Version is found but it's not up to date:**
`The extension cannot initialize because you are using the wrong version of the CLI`
Buttons: Setup

**Version is found but it's too far ahead:**
`The extension cannot initialize because you are using the wrong version of the extension.`
Button: Setup

What do we think? Should the toasts be more/less simplified?

Reply:
Updated Toasts:

**Version not found:**
`An error was thrown when trying to access the CLI.`
buttons: Setup, Don't Show Again

**Version can not be verified:**
`The extension cannot initialize as we were unable to verify the DVC CLI version.`
Button: Setup

**Version behind or ahead**
`The extension cannot initialize because the DVC CLI version is incompatible.`
Button: Setup